### PR TITLE
Add page for #tfp-jobs-for-palestinians project

### PR DESCRIPTION
**Pull request by NateWr**
This PR adds a page at `/jobs-for-palestinians` that lists organizations helping to connect Palestinian job seekers with employers. This is a draft, pending final review in [#tfp-jobs-for-palestinians](https://discord.com/channels/1186702814341234740/1204873076077109248) (Discord).

TODO:
- [x] Design review by the #tfp-jobs-for-palestinians team.
- [x] Code review by web team.
- [x] Two of the resources need more information before they can be added. See [Google Doc](https://docs.google.com/document/d/1lH6r6BzeOS9lWwsuhIRc7z98nl8B8H1HU2c4iRC48OY/edit) (not public).
- [x] Should this page be linked to from the main projects list?

### CODE REVIEW

**Comment by janajri** [comment]
@NateWr is this ready for review?

**Comment by NateWr** [comment]
@janajri no, some changes are currently being discussed in #tfp-jobs-for-palestinians (Discord).

File: src/pages/jobs-for-palestinians.astro [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+const resources = [
+  {
+    name: `Connect Palestine`,
+    url: 'https://join.slack.com/t/connectjobsfo-6xs9807/shared_invite/zt-2aklqmhci-3hb33MV~Bps5fKIV6dr9fg',
+    logo: '/slack.webp',
+    description: `A Slack platform to connect job seekers with employers, share job opportunities, provide mentorship and career advice, and establish partnerships and collaborations with pro-Palestinian companies.`,
+  },
+  {
+    name: 'Mena Alliances',
```
Comment (pbiggar) [inline src/pages/jobs-for-palestinians.astro:32]:
```suggestion
    name: 'MENA Alliances',
```

**Comment by pbiggar** [comment]
Can you include screenshots with PRs?

**Comment by NateWr** [comment]
@webzetetic sorry it's been a busy couple of days. I can do it at the weekend.

**Comment by NateWr** [comment]
PR updated to add categories as discussed in #tfp-jobs-for-palestinians (Discord). 

Here are screenshots on Ubuntu/Firefox (sorry, the Astro dev block is the dark button visible in each one -- ignore that):

tfp-job-desktop
tfp-job-laptop-sm
tfp-job-mobile

**Comment by pbiggar** [comment]
Looks great!

- I think "Mena Alliances" should be "MENA Alliances"
- Instead of "view" should it say "visit"

All good on my end otherwise. Asking @janajri to look at the code.

**Comment by NateWr** [comment]
Thanks @pbiggar. I've made your copyedit changes and added the Connect Palestine logo provided by a member of that group.

(fun fact: "Mena" is the British English convention for acronyms. "MENA" is the America English convention. See [Nasa](https://www.google.com/search?channel=fs&client=ubuntu-sn&q=site%3Aguardian.co.uk+nasa) vs [NASA](https://www.google.com/search?channel=fs&client=ubuntu-sn&q=site%3Anytimes.com+nasa).)

**Comment by janajri** [comment]
@NateWr is this ready for review?

**Comment by NateWr** [comment]
@janajri thanks, I've asked for final approve from the project lead

**Comment by NateWr** [comment]
@janajri the project lead says it's good to go. :+1:

**Pull request merged**